test/test_posixtest.py

```diff
@@ -166,6 +166,9 @@ def f(self):
             '-Wno-int-conversion',
             '-Wno-format',
             '-pthread',
+            # Make sure all tests have callstacks to improve debuggability
+            # of log messages on CI runs.
+            '--profiling-funcs',
```

Comment:
I've always thought it might be good to add this flag to more of out test suite (for example the core0 suite) but @kripken was worried about the testing then not matching to defaults used by out users.

Reply:
I did have that in mind, and I agree. That is why I specifically added `--profiling-funcs` and not `-g` here.

Though in this case the posixtest suite is more of a satellite suite, and there will still be `core0` suite that verifies builds without debug symbols. (which includes multithreading tests). It is very unlikely that a test in posixtest suite would mask a failure by having `--profiling-funcs`, that the `core0` suite wouldn't also have.